ntoskrnl/mm/ARM3/mminit.c

```diff
@@ -1911,7 +1914,7 @@ MiBuildPagedPool(VOID)
     //
     // We'll also allocate the bitmap header itself part of the same buffer.
     //
-    Size = Size * 1024;
+    Size = Size * PTE_COUNT;
     ASSERT(Size == MmSizeOfPagedPoolInPages);
     BitMapSize = (ULONG)Size;
     Size = sizeof(RTL_BITMAP) + (((Size + 31) / 32) * sizeof(ULONG));
```

Comment:
Can we also use some ROUND_UP macro here?

Reply:
Probably, feel free to do that, but please use ALIGN_UP_BY! ;-)